tools/prepare-onebranch-build.ps1

```diff
@@ -12,12 +12,12 @@ $RootDir = Split-Path $PSScriptRoot -Parent
 $BinDir = Join-Path $RootDir "artifacts\bin"
 $IncludeDir = Join-Path $RootDir "inc"
 
-$Configs = @("chk", "fre")
-$Platforms = @("amd64", "arm64")
+$Configs = @("Debug", "Release")
+$Platforms = @("x64", "arm64")
 
 foreach ($Config in $Configs) {
     foreach ($Platform in $Platforms) {
-        $VpackDir = Join-Path $BinDir "$($Platform)$($Config)"
+        $VpackDir = Join-Path $BinDir "$($Platform)_$($Config)"
```

Comment:
Are you sure this will actually work with the VPack requirement internally. I'm pretty sure the chk/fre stuff is an absolute requirement for them.

Reply:
Only one way to find out!

Reply:
not 100% sure but there's a staging step that copies file to amd64fre style folders.